doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/primitives/text/localization/TextLocalizationPage.swift

```diff
@@ -1,26 +1,49 @@
 import SwiftUI
 
 struct TextLocalizationPage: View {
-    var someString = "pen"
-    var object = LocalizedStringResource("pencil")
     
     var body: some View {
         ScrollView {
             VStack(spacing: 24) {
-                // Localized if possible
+                // Text's initializer accepts a LocalizedStringKey, which can be implicitly constructed from a string literal
+                // The value of the string is used for the development language localization
+                // Values for other languages are looked up in the default table using the string as key, if not found the value is used as is
                 Text("pencil")
                 
-                // Variables not localized by default
+                // The comment is shown in the localization table, useful for giving context to translators
+                Text("another pencil", comment: "Translator comment")
+                
+                // This comment will be ignored, since the same key was declared earlier. This is valid even if no comment was specified on the previous key.
+                Text("another pencil", comment: "A different comment")
+                
+                // Specify a different table for this localization
+                Text("pencil", tableName: "LocalizableAlternative")
```

Comment:
Are there any naming requirements for the table name? Do they have to be "LocalizableXXXX"?

Reply:
LocalizableXXX is the naming convention for string catalogs. The only requirement here is that it is the name of an existing string catalog.
I added to the comment to make this clearer